test/src/e2e_vm_tests/test_programs/should_pass/stdlib/u128_log_test/src/main.sw

```diff
@@ -0,0 +1,28 @@
+script;
+
+use std::{assert::assert, math::*};
+use std::revert::revert;
+use core::num::*;
+use std::u128::*;
+
+fn main() -> bool {
+    let u_128_0: U128 = ~U128::from(0, 0);
```

Comment:
These test are good but could be better. It would be nice to see some very large `U128` numbers here.

Reply:
Added such cases